sdk/documentintelligence/ai-document-intelligence-rest/review/ai-document-intelligence.api.md

```diff
@@ -1078,7 +1078,7 @@ export interface DocumentKeyValuePairOutput {
 // @public
 export interface DocumentLanguageOutput {
     confidence: number;
-    locale: string;
+    locale: number;
```

Comment:
References: `locala` -> `DocumentLanguageOutput ` -> `AnalyzeResultOutput ` -> `AnalyzeResultOperationOutput ` ->  `GetAnalyzeResult200Response ` and `GetClassifyResult200Response `, so anything used these responses get impacted. 
e.g. 
```ts
// @public (undocumented)
export function isUnexpected(response: GetClassifyResult200Response | GetClassifyResultDefaultResponse): response is GetClassifyResultDefaultResponse;
```